### PR TITLE
docs: Update installation instructions

### PR DESCRIPTION
**Pull request by luisfabib**

### CODE REVIEW

**Review by stestoll — APPROVED** [review]

File: docsrc/source/installation.rst [hunk 1]
```diff
@@ -69,15 +68,41 @@ Download and installation instruction for the MATLAB environment are provided in
 Installing from source
 ----------------------
 
-To install DeerLab from the source, downloaded or clone the source code from the `DeerLab repository <https://github.com/JeschkeLab/DeerLab>`_.
-DeerLab and its dependencies can be installed by running the following command on a terminal window to install DeerLab as a static Python package::
+If you wish to contribute to DeerLab or like to get the latest updates as they come, install DeerLab from ``git``. If your OS has not ``git`` installed, you can download it and install it from the `official Git distribution <https://git-scm.com/>`_.
+To download (clone) the repository, execute the following from the command line::
 
-		python -m setup.py install
+		git clone git@github.com:JeschkeLab/DeerLab.git
```
Comment (laenan8466) [inline docsrc/source/installation.rst:74]: Replace git statement with:
`git clone https://github.com/JeschkeLab/DeerLab.git `

File: docsrc/source/installation.rst [hunk 2]
```diff
@@ -69,15 +68,41 @@ Download and installation instruction for the MATLAB environment are provided in
...
+
+
+Installing from Anaconda
+-------------------------
+
+DeerLab is currently not distributed via the ``conda`` or Anaconda package managers. However, ``conda`` or Anaconda users can still install and/or update the software as follows. 
+
+From a terminal (preferably with administrative privileges) choose a local directory where the program should be stored and clone DeerLab from the source as described above :: 
+
+		git clone git@github.com:JeschkeLab/DeerLab.git
```
Comment (laenan8466) [inline docsrc/source/installation.rst:98]: Replace clone statement with:
`git clone https://github.com/JeschkeLab/DeerLab.git`

I‘m not sure how to handle the statement above for contributors.

Comment (luisfabib) [reply docsrc/source/installation.rst:98]: Good point! We need to change this!

**Comment by luisfabib** [comment]
Since we are now properly distributing DeerLab through the Anaconda repo (see #157) we need to adapt the installation instructions again.

**Review by stestoll — DISMISSED** [review]

**Comment by mtessmer** [comment]
What's the difference between phygbu memoization and conda-forge memoization?

**Comment by luisfabib** [comment]
If I understood correctly, the conda-forge channel `memoization` is the copy of the phygbu channel `memoization`.  

We could just change the install call to 
```
conda install deerlab -c conda-forge -c JeschkeLab
```

**Comment by mtessmer** [comment]
IOf using from conda-forge then you only need `conda install deerlab -c JeschkeLab`  Also it looks like the conda-forge version is more up to date (0.3.2 vs 0.3.1)

**Comment by luisfabib** [comment]
Great! I did not know that the conda-forge channel was by default included. I'll adapt the instructions.

**Review by stestoll — APPROVED** [review]

**Pull request merged**